### ISSUE TITLE
Non- Darwin Core Fields in the data download

### ISSUE THREAD

**Issue by albenson-usgs** [iobis/obis-issues#148]
I have a user and data provider, Stace Beaulieu, who was extremely confused by all the "extra" fields that come with the data download. To her it seemed like she had done something wrong in the download. We need some way to either have those non-Darwin Core fields as a separate download or we need those to be indicated clearly as being quality flags and WoRMS data for users. Ideally, we should allow users to choose to download those fields if they want them.

**Comment by pieterprovoost** [comment]
Yes, we need to make a better distinction between source data and added information. But I suggest to include the extra fields in the downloads by default, the WoRMS taxonomy for example makes the data a lot easier to work with for most users.

The first step will be to separate the source and added fields in the API response, once that's there we can modify the R package and mapper to optionally omit the added fields or to keep them in a separate table.
